trustfall_core/test_data/tests/valid_queries/fold_with_no_outputs_transform_and_filter_one_of.graphql.ron

```diff
@@ -0,0 +1,22 @@
+TestGraphQLQuery (
+    schema_name: "numbers",
```

Comment:
An explanatory comment here would go a long way toward explaining what this test case is about.

Also, consider a variation of this test case that would make it a bit more interesting: What if the largest number of `min_primes` is enough below the actual count that there's an opportunity to discard the result early when it's clear it won't ever match? Does our optimization kick in then? Do we already have a test case for this?

Reply:
This test case actually doesn't fit anywhere since we don't at all touch one_of in the optimization pr I'm about to do

Reply:
This test case has been repurposed to that^